src/hotspot/share/runtime/os.hpp

```diff
@@ -966,6 +966,8 @@ class os: AllStatic {
   };
 #endif // !WINDOWS
 
+  static address ucontext_get_pc(const ucontext_t* ctx);
+  static void    ucontext_set_pc(ucontext_t* ctx, address pc);
```

Comment:
This feels misplaced here (and probably won't compile on windows) since ucontext_t is POSIX. At the very least needs ucontext.h. But I would consider moving this to os_posix.

Reply:
I thought I tested it and it built fine on Windows - will take another look...

Reply:
Fixed.